index.html

```diff
@@ -144,17 +159,20 @@ <h1 class="aboutme">About me</h1>
           <div class="endline">
             <img src="images/endline.png" alt="line" />
           </div>
-          <ul class="lfs">
-            <li>
+          <div class="deskendline">
+            <img src="images/deskendline.png" alt="deskendline">
+          </div>
+          <div class="lfs">
```

Comment:
Your languages section is also not aligned properly for larger screens. Consider fixing it.
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/177906604-9b8bd2b2-d6bf-4ec9-9d79-d5b7309620c4.png) | ![image](https://user-images.githubusercontent.com/33268927/177906535-cf597c21-3f01-4e04-a1a4-e015a982b853.png) |